projects/schematics/src/migrations/2_0/constructor-deprecations/data/product-reviews.component.migration.ts

```diff
@@ -0,0 +1,27 @@
+import {
+  SPARTACUS_STOREFRONTLIB,
+  PRODUCT_REVIEWS_COMPONENT,
+  ANGULAR_CORE,
+  FORM_BUILDER,
+  ANGULAR_FORMS,
+  CHANGE_DETECTOR_REF,
+} from '../../../../shared/constants';
+import { ConstructorDeprecation } from '../../../../shared/utils/file-utils';
+
+export const PRODUCT_REVIEWS_COMPONENT_MIGRATION: ConstructorDeprecation = {
+  // projects\storefrontlib\src\cms-components\product\product-tabs\product-reviews\product-reviews.component.ts
+  class: PRODUCT_REVIEWS_COMPONENT,
+  importPath: SPARTACUS_STOREFRONTLIB,
+  deprecatedParams: [
```

Comment:
Deprecated params should also contain ProductReviewService and CurrentProductService